wordpress-mobile/gutenberg-mobile
# Upgrade React Native `0.71.11` - iOS changes

**Related PRs:**

* https://github.com/WordPress/gutenberg/pull/51386
* https://github.com/wordpress-mobile/WordPress-iOS/pull/20956
* https://github.com/wordpress-mobile/gutenberg-mobile/pull/5924

**To test:**
* Observe that all CI jobs pass, except "Build Android RN Aztec & Publish to S3". This failure will be incorporated into the RN upgrade in another separate PR.
* Optionally, follow testing instructions from https://github.com/WordPress/gutenberg/pull/51303.

PR submission checklist:
